packages/react/src/components/carbon-components-react/UIShell/HeaderMenu.js

```diff
@@ -87,8 +88,13 @@ class HeaderMenu extends React.Component {
    */
   handleOnClick = index => {
     this.setState(prevState => {
-      if (prevState.expanded) this.props.setOverlay(false);
-      else this.props.setOverlay(true);
+      if (prevState.expanded) {
+        this.props.setOverlay(false);
+        root.document.body.style.overflowY = 'auto';
```

Comment:
Would we be able to define a CSS class for `<body>`? I tend to avoid inline style for the sake of customizability (though I know it's rare for this case).